src/controller/org.controller/index.js

```diff
@@ -166,23 +164,21 @@ router.get('/org/:identifier',
   /*
   #swagger.tags = ['Organization']
   #swagger.operationId = 'orgSingle'
-  #swagger.summary = "Retrieve an organization's data given the identifier, which can be their short name or the organization's uuid."
+  #swagger.summary = "Retrieves the organization entry for the specified short name or UUID (accessible to all users)"
   #swagger.description = "
         <h2>Access Control</h2>
-        <p>No roles needed to access the endpoint</p>
+        <p>All registered users can access this endpoint</p>
         <h2>Expected Behavior</h2>
-        <p><b>Secretariat:</b> Can see the information of any Organization</p>
-        <p><b>Admin User:</b> Can only see the information of the Organization it belongs to</p>
-        <p><b>Regular User:</b> Can only see the information of the Organization it belongs to</p>
-  "
+        <p><b>Regular, CNA & Admin Users:</b> Retrieves organization record for the specified short name or UUID if it's the user's organization</p>
```

Comment:
Minor - would be better to say "it is" instead of "it's". Or we could say "if it matches the user's org"